### ISSUE TITLE
v4.4.4 Breaking Build

### ISSUE THREAD

**Issue by jspizziri** [nestjsx/crud#477]
@zMotivat0r ,

Just upgraded to fix the SQL injection issue in addition to fixing the issue with the Lowercase search filters and now I'm getting this on build:

```
 ❯ npm run build

> api@0.0.1 prebuild /app
> rimraf dist


> api@0.0.1 build /app
> nest build

node_modules/@nestjsx/crud/lib/types/query-filter-option.type.d.ts:1:41 - error TS2307: Cannot find module '@nestjsx/crud-request/src/types/request-query.types'.

1 import { QueryFilter, SCondition } from '@nestjsx/crud-request/src/types/request-query.types';
                                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Found 1 error(s).

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! api@0.0.1 build: `nest build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the api@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /app/.npm/_logs/2020-04-18T18_44_46_499Z-debug.log
```

**Comment by zMotivat0r** [comment]
will fix ASAP

**Comment by zMotivat0r** [comment]
fixed with v4.4.5
